py/mattak/backends/pyroot/mattakloader.py

```diff
@@ -2,6 +2,9 @@
 import sys
 import platform
 import os
+import logging
+logger = logging.getLogger(__name__)
+logging.basicConfig(level = logging.DEBUG)
```

Comment:
Doesn't that cause the logger to always be logging the debug messages when this script is called (and before `logging.basicConfig()` isn't called? I thin you can just remove that last line